src/components/calculator/calculator.js

```diff
@@ -1,19 +1,24 @@
 /* eslint-disable */
-import React, { Component } from 'react';
+import { useState } from 'react';
 import Header from '../header/header';
 import Button from './button/ButtonCard';
 import ButtonValue from './button/ButtonScreen';
 import './calculator.css';
 import calculate from './logic/calculate';
 
-export default class App extends Component {
-  state = {
+
+
+function calculator() {
```

Comment:
The app does not function as it should. The user is not able to perform any calculation. Moreover, kindly make the `dev` branch the default branch.